src/main/cpp/hw/DragonSparkFlex.cpp

```diff
@@ -41,6 +51,10 @@ DragonSparkFlex::DragonSparkFlex(int id,
     m_spark->SetClosedLoopRampRate(0.02);
     m_encoder.SetPosition(0);
     SetRotationOffset(0);
+    m_forwardLimitSwitch.EnableLimitSwitch(false);
+    m_reverseLimitSwitch.EnableLimitSwitch(false);
+    m_forwardType = rev::SparkLimitSwitch::Type::kNormallyOpen;
```

Comment:
Why are m_forwardType and m_reverseType being reassigned when they are given a value in the constructor?

Reply:
Agree removed